### PR TITLE
ABU suggested and RH Marketing committed changes

### PR DESCRIPTION
**Pull request by wbentley15**
Howdy,

This PR consists of numerous suggestions per my leadership and adjustments committed to RH Marketing required for launch.  Yes, there are a lot of changes so I apologize in advance for that.  Wanted to make sure we would be cleared for launch.  Certainly open to discussion on any adjustments.  Just note any deviations will cause further delays in our launch.  Goal is to get it out the door. 

Thanks,
Walter

### CODE REVIEW

**Comment by oraNod** [comment]
Thanks @wbentley15 

Yes, as you said, there are quite a few changes in here for a single PR. If I may suggest it, you should consider breaking these into atomic commits/PRs. There's a really good resource that goes into some detail about why atomic commits are preferred here: https://gist.github.com/webknjaz/a7362787a80067af8621a85a71746ca1

TL;DR - atomic commits/PRs are much easier to review and it isolates changes in a way that leads to a much clearer git history. There are many other benefits outlined in that link.

**Comment by wbentley15** [comment]
@oraNod I will be honest with you to say I do not have time to research the approach you provided.  Need to move on to my other responsibilities.  As mentioned, everything thing here is a requirement for launch.  Check in with @Spredzy on Monday as I have already spoken with him.  Have a great weekend!

**Comment by wbentley15** [comment]
FYI @oraNod I am aware the page named 'automation-adoption' is missing.  I am still drafting that article.  Goal is to have it into a PR next week.  Saw it was the failure of the nox build for this PR.

**Comment by oraNod** [comment]
> @oraNod I will be honest with you to say I do not have time to research the approach you provided.  Need to move on to my other responsibilities.  As mentioned, everything thing here is a requirement for launch.  Check in with @Spredzy on Monday as I have already spoken with him.  Have a great weekend!

will do @wbentley15 and don't worry. I'm here to help out. enjoy your weekend.

**Comment by wbentley15** [comment]
Thank you @oraNod and enjoy your weekend too!

**Comment by oraNod** [comment]
@wbentley15 Along with the CI fixes I think we need some changes for accessibility. Here is a link to the report for the current preview build: https://wave.webaim.org/report#/https://ansible-community-website--418.org.readthedocs.build/

It looks like there are some alerts for suspicious links, contrast errors, incorrect heading levels, and the embedded video:

image

**Comment by oraNod** [comment]
@wbentley15 Another thing to note is that the embedded video causes overflow issues on smaller displays when you go into responsive design mode.

image

File: .doit.db.db
Comment (oraNod) [inline .doit.db.db:1]: Was this added intentionally?

File: conf.py [hunk 1]
```diff
@@ -161,16 +161,21 @@
...
+            ("/faq/", "FAQs", ""),
             ("/ansible-community-training/", "Ansible community training", ""),
-            ("/mission-statement/", "Our mission statement", ""),
+            
             ("/contact-us/", "Contact us", ""),
           ),
           "Resources", ""
         ),
-        ("https://forum.ansible.com/", "Ansible community forum", ""),
+        ("https://forum.ansible.com/", "Forum", ""),
```
Comment (oraNod) [inline conf.py:176]: We explicitly chose "Ansible community forum" to make it clear that it is not somewhere RH customers should go to find support etc.

File: conf.py [hunk 2]
```diff
@@ -161,16 +161,21 @@
...
         ("https://docs.ansible.com/", "Documentation", ""),
+        ("https://www.redhat.com/en/technologies/management/ansible/", "Ansible Automation Platform", ""),
```
Comment (oraNod) [inline conf.py:178]: Should this be "Red Hat Ansible Automation Platform"?

File: data/homepage.yaml [hunk 3]
```diff
@@ -7,10 +7,20 @@ banner:
   redhat_logo_alt: Red Hat logo
   welcome_alt: Welcome to Ansible
   platform_learn: Learn about Red Hat Ansible Automation Platform
+events:
+  section_icon: /assets/images/calendar-cyan.svg
+  icon_alt: Ansible community event icon
+  title: Events
+  intro: 
+  url:
```
Comment (oraNod) [inline data/homepage.yaml:15]: It seems like the Events banner has been removed. Should we not just delete these fields if they are unused?

File: data/homepage.yaml [hunk 4]
```diff
@@ -7,10 +7,20 @@ banner:
...
+  intro: 
+  url: 
 ansible:
   section_icon: /assets/images/automation-cyan.svg
   icon_alt: Ansible automation icon
   image_alt: Ansible community collaboration
+  mission: Click here to learn more about our mission
+  mission_url: link://slug/mission-statement
+  join: Click here to join the community
+  join_url: https://docs.ansible.com/community.html
```
Comment (oraNod) [inline data/homepage.yaml:23]: These two are reported as suspicious links in violation of WCAG rules. Please see here for more info: https://webaim.org/standards/wcag/checklist#sc2.4.4

File: data/homepage.yaml [hunk 5]
```diff
@@ -165,3 +135,37 @@ links:
...
     use_cases: https://www.redhat.com/en/technologies/management/ansible/use-cases
+welcome:
+  title: Welcome to the new Ansible.com!
+  intro: Welcome home and thank you for supporting Ansible. View the video to learn about what changed and why...
+  url: https://www.youtube.com/embed/zHrIo4E1HmY?si=aXMEyR14P_6gKoj2
+catalyst:
+  section_icon: /assets/images/orchestration-cyan.svg
+  icon_alt: Ansible Catalyst program icon
+  title: Ansible Catalyst
+  intro: This is a new formed program lead by Red Hat to spark additional user interaction, spotlight the ecosystems contributions and share insights on driving automation adoption at your organizations.
```
Comment (oraNod) [inline data/homepage.yaml:146]: For grammatical correctness either "newly formed" or "new program".

I would also suggest changing "spark" to something else like "encourage" or maybe "foster". I think the word spark is too idiomatic here. Also "spotlight" is a noun but is being used as a verb. You might consider changing it to "highlight" or "showcase".

It's probably a good idea to add a comma before "and share insights".

File: data/homepage.yaml [hunk 6]
```diff
@@ -165,3 +135,37 @@ links:
...
+  articles:
+    collection_spotlight:
+      heading: Collection Spotlight
+      description: Every week we will spotlight a new collection available to the Ansible ecosystem.
```
Comment (oraNod) [inline data/homepage.yaml:150]: Again with the word "spotlight". It's somewhat overloaded here. It's fine in the heading but I would suggest using a synonym in the description.

File: data/homepage.yaml [hunk 7]
```diff
@@ -165,3 +135,37 @@ links:
...
+  intro: Welcome home and thank you for supporting Ansible. View the video to learn about what changed and why...
+  url: https://www.youtube.com/embed/zHrIo4E1HmY?si=aXMEyR14P_6gKoj2
+catalyst:
+  section_icon: /assets/images/orchestration-cyan.svg
+  icon_alt: Ansible Catalyst program icon
+  title: Ansible Catalyst
+  intro: This is a new formed program lead by Red Hat to spark additional user interaction, spotlight the ecosystems contributions and share insights on driving automation adoption at your organizations.
+  articles:
+    collection_spotlight:
+      heading: Collection Spotlight
```
Comment (oraNod) [inline data/homepage.yaml:149]: I'll make this a global comment. Please use sentence case for headings.

File: pages/faq.md [hunk 8]
```diff
@@ -0,0 +1,182 @@
+---
+slug: faq
```
Comment (oraNod) [inline pages/faq.md:2]: Is this FAQ for the entire site or just AWX?

File: pages/faq.md [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+[Q: How often is AWX released?](#10)  
+[Q: What is the governance structure of the AWX project?](#11)  
+[Q: If my software runs with AWX, can I say that it is certified to run on AWX or on Red Hat Ansible Automation Platform?](#12)  
+[Q: I want to build my own forked version of AWX. Can I call it AWX? Can I call it Red Hat Ansible Automation Platform?](#13)  
+[Q: Will contributions to AWX require a “Contributor License Agreement”?](#14)  
+[Q: Does Red Hat recommend AWX for production environments?](#15)  
+[Q: Does Red Hat’s Open Source Assurance Program apply to AWX?](#16)  
+**Ansible Community**  
+[Q: How can I contribute back to the community?](#17)  
+<br><br>
```
Comment (oraNod) [inline pages/faq.md:44]: These line breaks are unclosed tags.

File: pages/faq.md [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+
+<a name="16"><a>
+
+**Does Red Hat’s Open Source Assurance Program apply to AWX?**
+
+No.
+
+---
+
+# Ansible Community
```
Comment (oraNod) [inline pages/faq.md:166]: I guess this answers my previous question about the FAQ but this change really buries the community part in an old FAQ section. I don't know if I would expect to find this here.

File: templates/homepage-ansible-band.tmpl [hunk 11]
```diff
@@ -15,6 +15,8 @@
       <p>{{ homepage.ansible.top_section.body }}</p>
       <h3>{{ homepage.ansible.bottom_section.heading }}</h3>
       <p>{{ homepage.ansible.bottom_section.body }}</p>
+      <span class="homepage-link"><a href="{{ homepage.ansible.mission_url }}">{{ homepage.ansible.mission }}</a></span><br>
```
Comment (oraNod) [inline templates/homepage-ansible-band.tmpl:18]: br tag is unclosed

File: templates/homepage-catalyst-band.tmpl [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+      <h2>{{ homepage.catalyst.title }}</h2>
+    </div>
+    <p>{{ homepage.catalyst.intro }}</p>
+  </div>
+  {% for key, item in homepage.catalyst.articles.items() %}
+    <div class="catalyst-card width-6-12 width-12-12-m">
+      <div class="catalyst-content">
+        <h3 style="color: red;">{{ item.heading }}</h3>
+        <p>{{ item.description }}</p>
+        <h5>{{ item.data_title }}</h5>
```
Comment (oraNod) [inline templates/homepage-catalyst-band.tmpl:19]: From the accessibility report this skips a heading level. It goes from h3 to h5.

File: templates/homepage-catalyst-band.tmpl [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+           width="40"
+           height="40" />
+      <h2>{{ homepage.catalyst.title }}</h2>
+    </div>
+    <p>{{ homepage.catalyst.intro }}</p>
+  </div>
+  {% for key, item in homepage.catalyst.articles.items() %}
+    <div class="catalyst-card width-6-12 width-12-12-m">
+      <div class="catalyst-content">
+        <h3 style="color: red;">{{ item.heading }}</h3>
```
Comment (oraNod) [inline templates/homepage-catalyst-band.tmpl:17]: Please avoid inline styles.

File: templates/homepage-welcome-band.tmpl [hunk 14]
```diff
@@ -0,0 +1,15 @@
+<div class="grid-wrapper homepage-welcome-band content-slim"
+     id="welcome">
+  <div class="width-12-12 width-12-12-m">
+    <div class="welcome-row">
+      <div class="welcome-box-left">
+        <h2>{{ homepage.welcome.title }}</h2>
+        <p>{{ homepage.welcome.intro }}</p>
+      </div>
+      <div class="welcome-box-right max-width-prose">
+        <iframe width="420" height="300" src="{{ homepage.welcome.url }}">
```
Comment (oraNod) [inline templates/homepage-welcome-band.tmpl:10]: You could probably just add the closing tag here like this:

``<iframe width="420" height="300" src="{{ homepage.welcome.url }}" />``

File: themes/ansible-community/sass/_homepage-catalyst-band.scss [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+.catalyst-content {
+  display: flex;
+  flex-direction: column;
+  padding-right: 2rem;
+  justify-content: space-between;
+
+  h3 {
+    font-size: $font-lg;
+    font-weight: 600;
+    text-transform: none;
```
Comment (oraNod) [inline themes/ansible-community/sass/_homepage-catalyst-band.scss:25]: Move the font color from the inline style to here.

File: themes/ansible-community/sass/_homepage-community-band.scss [hunk 16]
```diff
@@ -6,7 +6,6 @@
   display: flex;
   flex-direction: row;
   width: 80%;
-  margin-top: 1rem;
```
Comment (oraNod) [inline themes/ansible-community/sass/_homepage-community-band.scss:9]: Why was this removed?

File: themes/ansible-community/sass/_homepage-welcome-band.scss [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+
+  .welcome-box-left {
+    padding-bottom: 2rem;
+  }
+
+  .welcome-box-right {
+    grid-column-start: initial;
+    grid-column-end: initial;
+  }
+}
```
Comment (oraNod) [inline themes/ansible-community/sass/_homepage-welcome-band.scss:43]: Please add an extra line at the end of the file.

File: .doit.db.db
Comment (wbentley15) [reply .doit.db.db:1]: No...must have come from Visual Studio.  Can be removed.

File: conf.py (hunk 1 above)
Comment (wbentley15) [reply conf.py:176]: This was an intentional and suggested change I prefer we uphold.  The label change will not promote or prevent customers from attempting to find support at the forum.  The link label as it is was commented at appearing unnecessarily redundant.

File: conf.py (hunk 2 above)
Comment (wbentley15) [reply conf.py:178]: No, the current label was approved by ABU Marketing.

File: data/homepage.yaml (hunk 3 above)
Comment (wbentley15) [reply data/homepage.yaml:15]: That is up to you as I thought it would be good to have in the event the banner is brought back.

File: templates/homepage-welcome-band.tmpl (hunk 14 above)
Comment (wbentley15) [reply templates/homepage-welcome-band.tmpl:10]: If you can implement that change for me I would appreciate it.  My focus is on other activities today.

File: themes/ansible-community/sass/_homepage-catalyst-band.scss (hunk 15 above)
Comment (wbentley15) [reply themes/ansible-community/sass/_homepage-catalyst-band.scss:25]: If you can implement that change for me I would appreciate it. My focus is on other activities today.

File: themes/ansible-community/sass/_homepage-community-band.scss (hunk 16 above)
Comment (wbentley15) [reply themes/ansible-community/sass/_homepage-community-band.scss:9]: No clue, must have been a fat finger by me.

File: themes/ansible-community/sass/_homepage-welcome-band.scss (hunk 17 above)
Comment (wbentley15) [reply themes/ansible-community/sass/_homepage-welcome-band.scss:43]: If you can implement that change for me I would appreciate it. My focus is on other activities today.

File: .doit.db.db
Comment (wbentley15) [reply .doit.db.db:1]: File removed.

File: data/homepage.yaml (hunk 5 above)
Comment (wbentley15) [reply data/homepage.yaml]: Changes applied.

File: pages/faq.md (hunk 8 above)
Comment (wbentley15) [reply pages/faq.md:2]: It is for the entire site.  I attempted to communicate the ask for this before and I think it got lost in the shuffle.  The suggestion was to have a stand alone FAQ page, of which I have created here.

File: pages/faq.md (hunk 9 above)
Comment (wbentley15) [reply pages/faq.md:44]: The <br> tag is an empty tag, which means that it has no end tag.  This is per HTML standards: https://www.w3schools.com/html/html_paragraphs.asp.  If you have a different understanding let me know.

File: templates/homepage-ansible-band.tmpl (hunk 11 above)
Comment (wbentley15) [reply templates/homepage-ansible-band.tmpl:18]: Same as stated above.

File: templates/homepage-catalyst-band.tmpl (hunk 12 above)
Comment (wbentley15) [reply templates/homepage-catalyst-band.tmpl:19]: The styling in place for heading was not consistent, so instead of changing them I just used the one that worked for the purpose needed.

File: templates/homepage-welcome-band.tmpl (hunk 14 above)
Comment (wbentley15) [reply templates/homepage-welcome-band.tmpl]: Nevermind. Change implemented.

File: themes/ansible-community/sass/_homepage-catalyst-band.scss (hunk 15 above)
Comment (wbentley15) [reply themes/ansible-community/sass/_homepage-catalyst-band.scss:25]: Nevermind.  Change implemented.

File: themes/ansible-community/sass/_homepage-community-band.scss (hunk 16 above)
Comment (wbentley15) [reply themes/ansible-community/sass/_homepage-community-band.scss:9]: Oh now I remember.  This band was too wide for the items being displayed and taking up unnecessary screen real estate.

File: themes/ansible-community/sass/_homepage-welcome-band.scss (hunk 17 above)
Comment (wbentley15) [reply themes/ansible-community/sass/_homepage-welcome-band.scss]: Nevermind.  The change was implemented.

File: data/homepage.yaml (hunk 3 above)
Comment (oraNod) [reply data/homepage.yaml:15]: Gotcha. This is why atomic commits - isolated changes - are preferred. In order to bring the event banner back it would be much easier to revert a single commit than to reconstruct from history.

In any case if we want to drop the event banner from the homepage, I would suggest simply removing the include directive from the main homepage template.

File: data/homepage.yaml (hunk 4 above)
Comment (oraNod) [reply data/homepage.yaml:23]: I believe it's the "click here" wording that triggers the WCAG alert. It's vague about where users are going and makes it harder for users with assistive tech like screen readers. This resource seems to have a good explanation: https://www.linkedin.com/pulse/main-reasons-why-we-shouldnt-use-click-here-link-text-toby-osbourn

File: pages/faq.md (hunk 9 above)
Comment (oraNod) [reply pages/faq.md:44]: Maybe it's just a tick that I got from somewhere - probably from my days of xhtml transforms - but I was referring to `<br />` 

I always use the closing forward slash like that but maybe it's just an old habit. I'm not an HTML guru or anything.

File: themes/ansible-community/sass/_homepage-community-band.scss (hunk 16 above)
Comment (oraNod) [reply themes/ansible-community/sass/_homepage-community-band.scss:9]: I would prefer to consult with @tiyiprh on some of these small changes. We spent a lot of time with the margins and padding to make things consistent and balanced.

Can we revert this specific change because it is unrelated to the others then create a separate PR?

**Comment by oraNod** [comment]
@wbentley15 It looks like I can't push directly to your branch so I opened a PR that you can merge to resolve some of the accessibility issues and CI checks: https://github.com/wbentley15/community-website/pull/1

**Comment by wbentley15** [comment]
@oraNod Yes, we can absolutely pivot from red to cyan for the heading color.  Hope that makes adjusting the color contrasts easier.

File: themes/ansible-community/sass/_homepage-community-band.scss (hunk 16 above)
Comment (tiyiprh) [reply themes/ansible-community/sass/_homepage-community-band.scss:9]: Let me know if you need anything specific from me on this!

**Comment by oraNod** [comment]
@wbentley15 Hey, one more PR for you to merge into your branch: https://github.com/wbentley15/community-website/pull/2

I've set the font color to cyan for the catalyst section, which resolves the accessibility issue with color contrast.

I also added a title to the embedded video for accessibility. We still get an alert about the video but I've verified that captions are available so I think we're good in terms of actually being accessible. You can find the report for that pull request here:

https://wave.webaim.org/report#/https://oranod-website.readthedocs.io/en/latest/

Finally I modified the welcome video band so it is more responsive. When the display resizes to 992px the text and video stack on top of each other. When the display hits 430px the video resizes to 300px to avoid overflow and when the display hits 300px we just hide the welcome band.

I don't think anyone is going to be viewing the site on a device with a 300px width or lower display but I thought it seemed reasonable to just hide the video at that stage because it avoids issues and the site text is still usable. We can change it again if needed though.

File: data/homepage.yaml [hunk 18]
```diff
@@ -165,3 +135,33 @@ links:
...
+      description: Have questions about how you can expand adoption of automation with your organizations?  If so, check out the following article to learn some tips and tricks.
+      data_title: Building your own automation community
+      data_body: Creating and supporting successful communities is something Red Hat has done for a long time.  We would love to share our framework with you on how we do it.
+      link: 
+    f2e:
+      heading: Enterprise automation
+      description: Curious about the journey from AWX to the Ansible Automation Platform?
+      data_title: When should an organization consider upgrading to expand their enterprise-grade automation strategy.
+      data_body: This article will help you define your organizations automation maturity and aid in determining if it is time to uplevel your strategy.
+      link: link://slug/enterprise-grade-strategy
```
Comment (oraNod) [inline data/homepage.yaml:167]: @wbentley15 This link is causing the CI failure with the nox build. Do you want to remove this link for now and add it in a follow up PR or do you plan to add an "enterprise automation" page as part of this PR?

**Comment by oraNod** [comment]
@wbentley15 One more for you. Hopefully you'll agree with the slight changes to the wording on some of those catalyst fields. Just let me know if you have any nits and we can work through them.

After that the only other thing I see here that might need tweaking is the fact the alternate band colouring got messed up when the welcome band dropped. Now the tab menu and the Ansible band both have the same background colour:

image

It also looks odd to have the platform band on top of the bullhorn band. That means there are two bands on top of each other without the "Back to top" link, which sort of breaks the navigation a little.

Another thing with the bullhorn band at the bottom is that it creates a narrow band of white between just before the footer on mobile:

image

We can play around with this some but what if we either restore the bullhorn band to the previous order - or move the platform band to where the bullhorn was previously?

**Comment by oraNod** [comment]
Sorry forgot to put the link to the pull request for your branch: https://github.com/wbentley15/community-website/pull/3
